### ISSUE TITLE
Language switcher is missing in block editor

### ISSUE THREAD

**Issue by g4rf** [ahmedkaludi/wp-multilang#14]
Hello and thanks for starting to maintaine the plugin again. As a problem I ran into, the language switcher is not shown anymore in the block editor, as the block editor core changed and the javascript hack of Valentyn do not work anymore. I rewrote the hack in [my own fork](https://github.com/g4rf/wp-multilang).

Nevertheless: Both, Valentyn's solution and mine, are javascript hacks and there must be something like a button API in the block editor, that should be the better approach.

**Comment by Sanjeevsetu** [comment]
Hi @g4rf 

I can see Valentyn's and yours javascript hacks solution are still working. Can you cross check it once??. And we are yet unable to find any API or hooks in block editor. As soon as We find it, We will implement.

**Comment by g4rf** [comment]
Hi @Sanjeevsetu 

thanks for digging into this issue. With the current version 2.4.5 the language switcher is (sometimes) not shown in posts:

![grafik](https://github.com/ahmedkaludi/wp-multilang/assets/18557074/d451953c-7f44-484d-95f3-de0acbe4697b)

It's because of a race condition: The javascript snippet tries to loads the language switcher, but the gutenberg toolbar isn't loaded yet. That's why I rewrote the javascript snippet with a deferred loading mechanism.

I opened the pull request #50 to solve this issue.
